bootstrap.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=utf-8
+import argparse
+import os
+import subprocess
+from string import Template
+
+from blessings import Terminal
+
+
+DEFAULT_DIRECTORY_NAME = 'mozilla-payments'
+GIT_REPOS = [
+    {'org': 'Kinto', 'name': 'kinto'},
+    {'org': 'mozilla', 'name': 'payments-example'},
+    {'org': 'mozilla', 'name': 'payments-service'},
+    {'org': 'mozilla', 'name': 'payments-ui'},
+    {'org': 'mozilla', 'name': 'solitude'},
+    {'org': 'mozilla', 'name': 'solitude-auth'},
+]
```

Comment:
Completely off topic so you can ignore, but I see this repeatedly, perhaps we can put this into docker-compose.yaml and have it read by docker-utils.